shared/teams/team/settings/retention/index.js

```diff
@@ -0,0 +1,187 @@
+// @flow
+import * as React from 'react'
+import {globalColors, globalMargins, globalStyles, isMobile, platformStyles} from '../../../../styles'
+import {Box, ClickableBox, Icon, ProgressIndicator, Text} from '../../../../common-adapters'
+import {type MenuItem} from '../../../../common-adapters/popup-menu'
+import {type RetentionPolicy, type _RetentionPolicy} from '../../../../constants/types/teams'
```

Comment:
why expose this component to the inner type and not just the record

Reply:
this component creates/holds a policy in it's state, and I thought it'd be overkill for it to use the record for that. In the second PR I got rid of the record and passed down JS in the prop. do you think I should make it use the record instead?

Reply:
i think a record or js is better but the inner type i think shouldn't really leak out

Reply:
ok, moved it to use just the record in #11094